### PR TITLE
octant-plugin cleanups

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

- Remove `octant-plugin` binary the repository :stuck_out_tongue_closed_eyes: 
- Enable *tests* on `octant-plugin`

/cc @ImJasonH 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/experimental/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by vdemeester** [comment]
I need to fix the CI :joy_cat:

File: test/presubmit-tests.sh [hunk 1]
```diff
@@ -35,4 +35,5 @@ function run() {
     return $exited
 }
 
-run webhooks-extension $@ || exit 1
\ No newline at end of file
+run webhooks-extension $@ || exit 1
+run octant-plugins $@ || exit 1
```
Comment (ImJasonH) [inline test/presubmit-tests.sh]: Should this be `octant-plugin`?

Comment (vdemeester) [reply test/presubmit-tests.sh]: Indeed 🤣

**Pull request merged**